nodejs modules (#2118)
kamranahmedse/developer-roadmap: content/roadmaps/107-nodejs/content/101-nodejs-modules/readme.md

> # Nodejs Modules

Reviewer: ```suggestion
# Node.js Modules
```

> # Nodejs modules
> 
> CommonJS modules are the original way to package JavaScript code for Node.js. Node.js also supports the ECMAScript modules standard used by browsers and other JavaScript runtimes, they are the official standard format to package JavaScript code for reuse.
> 
> Modules are defined using a variety of import and export statements. The two standards are not compatible, and Node.js provides a way to use both.
> 
> <ResourceGroupTitle>Free Content</ResourceGroupTitle>
> <BadgeLink colorScheme='blue' badgeText='Official Docs' href='https://nodejs.org/api/modules.html#modules-commonjs-modules'>Modules: CommonJS modules</BadgeLink>

Reviewer: ```suggestion
# Nodejs Modules

CommonJS modules are the original way to package JavaScript code for Node.js. Node.js also supports the ECMAScript modules standard used by browsers and other JavaScript runtimes, they are the official standard format to package JavaScript code for reuse.

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink colorScheme='blue' badgeText='Official Docs' href='https://nodejs.org/api/modules.html#modules-commonjs-modules'>Modules: CommonJS modules</BadgeLink>
href='https://blog.logrocket.com/commonjs-vs-es-modules-node-js/'>CommonJS vs. ES modules in Node.js</BadgeLink>
```
